### DISCUSSION TITLE
Disabling the toolbar

### DISCUSSION THREAD

**Discussion by glow12121** [gsantner/markor#2459] [Q&A] [answered]
Is it possible to disable the toolbar showing up above the keyboard in the editing mode? I type everything manually anyway, have a small screen and the bar merely takes space.

Thanks.

**Comment by harshad1** [comment, marked as answer]
Try to disable all the options for the given format? iirc that disables the bottom bar too/

**Reply by glow12121** [reply]
It works, thanks, Harshad!
